### DISCUSSION TITLE
Please a new version number!

### DISCUSSION THREAD

**Discussion by geraldFS** [omersiar/esp-rfid#585] [General]
Hello matjack1, hello everyone!

I think it's so cool that the project is finally getting going again and the project has finally got new super cool features.
And it goes on.

Since matjack1 version 2 is very stable despite the "beta status", I think it's finally time to upgrade to a new major version.

It deserves version number 2
Then version chaos will be better again.

Keep it up!
Gerald

**Comment by matjack1** [comment]
hey @geraldFS thank you very much for the nice words!!

I'm slowly working towards the V2: https://github.com/esprfid/esp-rfid/milestone/2

It's getting there, I'm about to close one issue and I have ideas for a few more open issues.

I'm skeptical about releasing a v2 with such important issues still open as I'm afraid that I will ruin the reputation of the project.

I know that there are a lot of new features on the current v2, but it's still quite unstable on certain use cases. I really hope to close this very soon though :)

Whoever can and is willing to contribute is very welcome!

**Comment by geraldFS** [comment]
Unfortunately, I can get involved because I don't have enough knowledge about programming.

I see myself more as a tester and evaluator.

I have a different opinion on this, the project has now been idle for a very long time, many people have left because there were no innovations.
It is now that it has taken a big step forward - the direction clearly visible!
So I think it's time for a "2.0 preview" - yes, there may be problems, but that can be communicated.
And it doesn't have much to do with the old 1.x anymore.

I also have wishes for the project - but the core function is there!

**Reply by matjack1** [reply]
I see your point, let's see if anyone else has opinions. Meanwhile I'll be working towards closing at least a few more bugs before releasing v2

**Reply by geraldFS** [reply]
Thanks very much!

**Comment by Leissson** [comment]
Agree with @geraldFS. It deserves v2 status. I've been using ESPHome integration for a while and waited websocket instability issues to be solved here.  Thank you @matjack1 for that.
